core/src/main/java/com/radixdlt/consensus/sync/BFTSync.java

```diff
@@ -427,21 +428,29 @@ private void sendBFTSyncRequest(
 
     if (syncRequestState.syncIds.isEmpty()) {
       if (this.syncRequestRateLimiter.tryAcquire()) {
-        VertexRequestTimeout scheduledTimeout = new VertexRequestTimeout(request);
-        this.timeoutDispatcher.dispatch(scheduledTimeout, bftSyncPatienceMillis);
+        this.timeoutDispatcher.dispatch(new VertexRequestTimeout(request), bftSyncPatienceMillis);
         this.requestSender.dispatch(authors.get(0), request);
       } else {
-        log.warn(
-            String.format(
-                "RATE_LIMIT: Outbound BFT Sync request %s for round %s due to %s to %s was not sent"
-                    + " because we're over our %s/second rate limit on sync requests",
-                request, round, reason, authors.get(0), this.syncRequestRateLimiter.getRate()));
+        // Report issue. Once per second as info-level message, rest as debug
+        if (loggingRateLimiter.tryAcquire() && log.isInfoEnabled()) {
+          log.info(buildMessage(reason, round, authors, request));
+        } else if (log.isDebugEnabled()) {
+          log.debug(buildMessage(reason, round, authors, request));
+        }
       }
       this.bftSyncing.put(request, syncRequestState);
     }
     syncRequestState.syncIds.add(syncId);
   }
 
+  private String buildMessage(
```

Comment:
I'd rename it to something more specific, e.g. `outboundRateLimitLogMessage`

Reply:
And pass an actual author as a parameter, rather than the whole list. It's not this method's responsibility to know that `authors.get(0)` was used.

Reply:
Good point. Renamed.

Reply:
Extracted local variable as it is used in at least 3 places.